src/commcare_cloud/terraform/modules/logshipping/main.tf

```
@@ -1,5 +1,7 @@
 locals {
   log_bucket_name = "dimagi-commcare-${var.environment}-logs"
+  log_bucket_prefix = "frontend-logs-${var.environment}"
+  log_bucket_error_prefix = "frontend-logs-${var.environment}-error"
```

This is going to be the log bucket prefix we use for storing the formplayer request/response logs, right? I'd change these to

```suggestion
  formplayer_request_response_log_bucket_prefix = "formplayer-request-response-logs-${var.environment}"
  formplayer_request_response_log_bucket_error_prefix = "formplayer-request-response-logs-${var.environment}-error"
```

you'd of course then have to rename the references below as well.

---

Ahh, is this just going to be used for formplayer or are we going to store other things in the future? I thought `logshipping` was pretty generic to other logs as well.

---

Yeah, looks like we will be using this module is a catch-all for logshipping that doesn't have another home, but still each individual usage of it is going to need to have its own write location in S3, so it'll need its own prefix and its own firehose stream (if using kinesis firehose).

---

Right, but then does it make sense to name have `formplayer` in the name if it's going to be used for other things as well? `frontend` was a copy and paste oversight, should we be passing these in as vars?